tuned.spec

```
@@ -299,7 +302,7 @@ to TuneD from power-profiles-daemon (PPD).
 make html %{make_python_arg}
 
 %install
-make install DESTDIR="%{buildroot}" BINDIR="%{_bindir}" SBINDIR="%{_sbindir}" \
+make install INSTALL_GUI="%{?gui}" DESTDIR="%{buildroot}" BINDIR="%{_bindir}" SBINDIR="%{_sbindir}" \
```

You should use the `%(with_foo}` inline macro, with `%bcond_{with,without} foo` and `%{with foo}` / `%{without foo}`.
```suggestion
make install %{!?with_gui:INSTALL_GUI=0} DESTDIR="%{buildroot}" BINDIR="%{_bindir}" SBINDIR="%{_sbindir}" \
```

---

Might even be better to have a `DISABLE_GUI` option instead, no default value, and `ifndef` that in the Makefile. `ifeq` is a pain when testing optional configs.